test/e2e/tests/account-details.spec.js

```diff
@@ -205,4 +205,134 @@ describe('Show account details', function () {
       },
     );
   });
+  it('should pin the account when pinned', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver }) => {
+        await unlockWallet(driver);
+
+        await driver.clickElement('[data-testid="account-menu-icon"]');
+        await driver.clickElement(
+          '[data-testid="account-list-item-menu-button"]',
+        );
+        await driver.clickElement('[data-testid="account-list-menu-pin"]');
```

Comment:
Can we extract them as a small helper function, namely `pingAccountItem` ?

Reply:
Hmmm...We can but for two cases we have this same function and then it's hiding and for some cases we don't have to define pinnedIcon again. So, I thought we can keep it like this. Wdyt?